### DISCUSSION TITLE
Option to disable Geode

### DISCUSSION THREAD

**Discussion by Elip100** [geode-sdk/sdk#1031] [Geode Ideas]
From what I can tell, there is no way to just disable Geode entirely. There is no option in-game, and when I set `should-load-geode.loader` to false in my save file, nothing changed. The only thing I can find is you can remove `geode.dll` from your game folder, but that still shows a popup before launching the game.

**Comment by matcool** [comment]
You can uninstall geode by uninstalling the geode mod ingame

**Reply by Elip100** [reply]
What I mean is a way to toggle it off without having to uninstall it. If I wanted to play vanilla GD for a short time, and then play modded, I would have to individually disable all of my mods. I feel like there should be an option to disable Geode itself, just like you can disable individual mods without uninstalling them.

**Reply by iNUCi** [reply]
Just open in safe mode

**Reply by Elip100** [reply]
> Just open in safe mode

bruh

**Reply by thatonecarinceleste** [reply]
i agree we need this

**Reply by HenrySck075** [reply]
> > Just open in safe mode
> 
> bruh

Just open in safe mode whats the problem

**Reply by thatonecarinceleste** [reply]
htf dyd that

**Reply by HenrySck075** [reply]
> htf dyd that

hold shift and open gd on pc and hold launch on android

**Reply by thatonecarinceleste** [reply]
thanks pookie

**Reply by Elip100** [reply]
> > htf dyd that
> 
> hold shift and open gd on pc and hold launch on android

Oh. I thought you meant windows safe mode XD
